src/scripts/Player.ts

```diff
@@ -45,6 +46,10 @@ class Player {
         }
         this._region = ko.observable(savedPlayer._region);
         this._subregion = ko.observable(savedPlayer._subregion || 0);
+        this.subregionObject = ko.observable(SubRegions.getSubRegionById(this._region(), this._subregion()));
+        this._region.subscribe(r => this.subregionObject(SubRegions.getSubRegionById(r, this._subregion())));
+        this._subregion.subscribe(sr => this.subregionObject(SubRegions.getSubRegionById(this._region(), sr)));
+
```

Comment:
You can use a pureComputed to automatically subscribe to these

```suggestion
        this.subregionObject = ko.pureComputed(() => SubRegions.getSubRegionById(this._region(), this._subregion()));
```

Reply:
I have committed your suggestion, and it still works!